src/app/_mermaid_common/forms/directives/ddlinput.tpl.html

```diff
@@ -3,8 +3,8 @@
     <span find-tooltip="ngModel[widgetName]"></span>
     <span ng-if="widgetRequired === true && widgetRequiredHide !== true" class="required" title="Required"> *</span>
   </label>
-  <div ng-class="{true: 'modal-trigger-on', false: ''}[modalController != null && mode == 'single']">
-    <label class="select{{ checkSymbol }}" ng-switch="choiceValue()" ng-class="{false: 'state-error', true: ''}[widgetForm[widgetName].$pristine || widgetForm[widgetName].$valid]">
+  <div ng-class="{true: 'modal-trigger-on', false: ''}[modalController != null && mode == 'single']" class="datasharing-dropdown">
```

Comment:
Not sure but seems to me we shouldn't have anything specific to a particular page/form in a directive